CMakeLists.txt

```diff
@@ -252,7 +254,7 @@ set_target_properties(${PROJECT_NAME} PROPERTIES
     SOVERSION
         ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}
     CXX_STANDARD
-        11
+        14
```

Comment:
Needed for some C++ feature used by the Prometheus library

Reply:
I am a little bit concerned about this but if the compiler did not bring any issues that may be ok.